doc/api/cli.md

```diff
@@ -320,6 +327,64 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+### `--report-directory`
+<!-- YAML
+added: REPLACEME
+-->
+
+Location at which the report will be generated.
+
+### `--report-filename`
+<!-- YAML
+added: REPLACEME
+-->
+
+Name of the file to which the report will be written.
+
+### `--report-on-fatalerror`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables the node-report to be triggered on fatalerror (internal errors within
+the Node.js runtime such as out of memory) that leads to termination of the
+application, if `--experimental-report` is enabled. Useful to inspect various
+diagnostic data elements such as heap, stack, event loop state, resource
+consumption etc. to reason about the fatalerror.
+
+### `--report-on-signal`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables report to be generated upon receiving the specified (or predefined)
+signal to the running Node process, if `--experimental-report` is enabled.
```

Comment:
This should mention that the signal is specified by `--report-signal` (and in that case listing the default both here and there might be superfluous)